osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -30,6 +37,9 @@ public class OsuPerformanceCalculator : PerformanceCalculator
         private int countMeh;
         private int countMiss;
 
+        private const double streamaimconst = 2.42;
+        private const double stdevconst = 0.149820;
```

Comment:
what are these magic constants? any explanation whatsoever?

Reply:
On this particular file it is not needed, already removed in 0cb6464.